### PR TITLE
How to install LEMP Stack + phpMyAdmin on Ubuntu 22.04

### PR DESCRIPTION
**Pull request by ssnikolas**

### CODE REVIEW

**Review by ssnikolas — COMMENTED** [review]
Some bug fix

**Comment by svenja11** [comment]
Thank you for your contribution @ssnikolas 

I looked at your tutorial and there already is a [LEMP stack tutorial for Ubuntu](https://community.hetzner.com/tutorials/install-lemp-stack-on-ubuntu-20.04) available at [community.hetzner.com](https://community.hetzner.com/tutorials/).

Please note the following contribution rules for tutorials:

> Before you start writing your tutorial, make sure there isn't already a tutorial at community.hetzner.com about the exact same topic. (source [Hetzner Tutorial Guidelines](https://github.com/hetzneronline/community-content/blob/master/contributing.md#content))

The existing tutorial does not cover the final two steps of your tutorial about phpMyAdmin and PHP configuration, so this part is fine.

However, for the installation of the LEMP stack, there should just be a link to the existing tutorial as suggested in the "Prerequisites" section of our tutorial template:

> If there is already a tutorial explaining one of the prerequisites, make sure to link to that other tutorial. (source [tutorial template](https://github.com/hetzneronline/community-content/blob/master/tutorial-template.md#introduction))

-------------------

I have updated your tutorial so that it links to the existing tutorial. In addition, I added the new section `Click here if you get a password error` in "Step 3 - Install phpMyAdmin" for readers who used the linked tutorial to install the LEMP stack.

I hope the changes I made are fine with you. Let me know if you have any questions!

File: tutorials/install-phpmyadmin-on-ubuntu/01.en.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+(c) The contribution was provided directly to me by some other person
+    who certified (a), (b) or (c) and I have not modified it.
+
+(d) I understand and agree that this project and the contribution are
+    public and that a record of the contribution (including all personal
+    information I submit with it, including my sign-off) is maintained
+    indefinitely and may be redistributed consistent with this project
+    or the license(s) involved.
+
+Signed-off-by: Mykola Shostakov
```
Comment (svenja11) [inline tutorials/install-phpmyadmin-on-ubuntu/01.en.md:233]: Please add your email address in the license block.

**Comment by svenja11** [comment]
Please also add this comment to your pull request:

I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: [submitter's name and email address here]

**Comment by svenja11** [comment]
Closing as there was no activity for a very long time.  
If you find time to keep working on this tutorial in future, feel free to open a new pull request.

**Pull request closed without merging**